content/docs/04-clusters/03-edge/02-edge-configuration/30-cloud-init.md

```diff
@@ -59,6 +59,63 @@ You may ask yourself where to use cloud-init stages, as both the Edge Installer
 
 * If you have common configurations across a fleet of Edge host devices, customize the OS pack and use the cloud-init stages to apply those configurations. 
 
+# Sensitive Information in the User Data Stages
+
+Suppose you plan to use sensitive information in your user data stages. In the edge deployment lifecycle, you can apply user data to edge hosts during the staging and site installation steps, as highlighted in the diagram below.  
+
+![A diagram highlighting the two stages in the edge deployment lifecycle where you can apply user data.](/edge_edge-configuration_cloud-init_user-data.png)
```

Comment:
In the image text. Change "Includes a supplementary user data" to "Supports adding supplementary user data that is site specific"